### DISCUSSION TITLE
wider styles dropdown box

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#451] [Ideas]
I suggest that the styles dropdown on the toolbar be widened in order to let longer named styles be easier to read. It appears that there is a lot of unused real estate on the toolbar that would allow this to be done. If there are no plans for filling up that space, could the dropdown be widened?

~ Bal

![Snag_7c7ffca3](https://user-images.githubusercontent.com/8197883/173207809-79b3ec28-8a95-4eae-86f2-3017bc2f0b90.png)

**Comment by rexelbartolome** [comment]
I've seen that Dimitry already made the size of the dropdown to be adaptive to the longest named style in the latest preview

https://user-images.githubusercontent.com/88552647/174464419-5de76eff-15b3-48c6-9213-d9ef02ec9461.mp4

**Reply by wordmuse** [reply]
Very cool! Much appreciated!
